### PR TITLE
#163: Add Proposed Solution Seed Data

### LINKED ISSUE #163: Backend - Add proposed solution seed data
### Description

current database has no proposed solution seed data for change requests. lets add some good examples to enhance the dev experience

### Background

.

### Proposed Solution

1. look at `src/backend/src/prisma/seed-data`
2. add some proposed solutions to a few of them (maybe add 1 to one, 2 to another, and 3 to another)

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by Danwo0**
## Changes

There are no seed data for proposed solutions. Let's change that.

## Screenshots (if applicable)

From prisma studio:
image

## Checklist

- [x] All commits are tagged with the ticket number
- [x] No linting errors
- [x] No newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (if applicable)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] PR is linked to the ticket
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack

Closes #163

### CODE REVIEW

**Review by jamescd18 — CHANGES_REQUESTED** [review]
Data generally looks good, but there's some tweaks in the insertion script I think are probably needed, and some notes for future:

1. no, you don't need to worry about the IDs for proposed solutions. Prisma handles generating those and so far I'm not aware of seed data needing to do anything with those IDs
2. knowing SQL is great and powerful, but you definitely wanna be sure to not overly rely on it. SQL implementation of the database can sometimes take you away from what's accessible / available / important at the application logic layer if you're not actively thinking about it. also [prisma studio](https://www.prisma.io/studio) comes with usage of prisma
3. seed data is intended to be used and thought of in a sorta closed system of all the seed data. when you seed or re-seed the db, you're intentionally wiping the db clean and loading it with your specified data. that way, typically, you can do some things that you wouldn't do otherwise

File: src/backend/src/prisma/seed.ts [hunk 1]
```diff
@@ -108,6 +109,25 @@ const performSeed: () => Promise<void> = async () => {
...
+    });
+
+    await prisma.proposed_Solution.create({
+      data: {
+        description: seedProposedSolution.description,
+        timelineImpact: crData?.timelineImpact ?? 0,
+        scopeImpact: crData?.scopeImpact ?? '',
+        budgetImpact: crData?.budgetImpact ?? 0,
+        changeRequestId: seedProposedSolution.changeRequestId,
+        createdByUserId: seedProposedSolution.createdByUserId,
```
[the remark this answers is not in the document]
Comment (anthonybernardi) [reply src/backend/src/prisma/seed.ts:125]: only if you're connecting by objects. if you connect by ids you can just hook up the ids like this

**Comment by anthonybernardi** [comment]
on the right track! but instead of doing all this connecting with the crData, you should just make the proposed solution seed objects have the full data (scope, budget, timeline) on their own. then you just have to make the proposed solution (`await prisma.proposed_solutions.createMany`) with that data. Also all the data should be meaningful (so try to think of stuff instead of just writing "example 1")

**Review by anthonybernardi — APPROVED** [review]
nice! LGTM

**Pull request merged**